stdlib/public/Python/Python.swift

```diff
@@ -1257,6 +1257,30 @@ extension PythonObject : MutableCollection {
   }
 }
 
+extension PythonObject : Sequence {
+  public struct Iterator : IteratorProtocol {
+    fileprivate let pythonIterator: PythonObject
+
+    public func next() -> PythonObject? {
+      guard let result = PyIter_Next(self.pythonIterator.borrowedPyObject) else {
+        try! throwPythonErrorIfPresent()
+        return nil
+      }
+
+      defer {
+        Py_DecRef(result)
+      }
+
+      return PythonObject(consuming: result)
+    }
+  }
+
+  public func makeIterator() -> Iterator {
+    let result = PyObject_GetIter(self.borrowedPyObject)
```

Comment:
Our local convention is to omit `self.` unless it's required.

```suggestion
    let result = PyObject_GetIter(borrowedPyObject)
```